### PR TITLE
Fixed comment encoding

### PR DESCRIPTION
**Pull request by la0wai**
Fixes the windows build problem discussed in issue #3552 

Machine translated some of the comments from Russian and saved as UTF8.
Some of the comments were totally boned so I replaced them with messages saying as such because I don't know C++ and couldn't hazard a guess as to what the functions actually do.

### CODE REVIEW

File: src/common/kernel.cpp [hunk 1]
```diff
@@ -92,7 +92,7 @@ sigfunc *compat_signal(int signo, sigfunc *func)
 
 /************************************************************************
 *                                                                       *
-*  CORE : Magical backtrace dump procedure (ﾉ◕‿◕)ﾉ*:･ﾟ✧                 *
+*  CORE : Magical backtrace dump procedure                              *
```
Comment (takhlaq) [inline src/common/kernel.cpp]: rip

File: src/common/socket.cpp [hunk 2]
```diff
@@ -215,7 +215,7 @@ int32 makeConnection(uint32 ip, uint16 port, int32 type)
 	return fd;
 }
 
-//�������, ����������� �����
+// the comment here was badly encoded and died.
```
Comment (takhlaq) [inline src/common/socket.cpp]: close socket

**Comment by teschnei** [comment]
hmm.. can you just remove them all instead?

I don't think the machine translations help, and it does seem like they were just stating the obvious in the first place

File: src/map/items/item_fish.h [hunk 3]
```diff
@@ -30,7 +30,7 @@
 
 /************************************************************************
 *                                                                       *
-*  ������� ��������� ���� ��� ���������� ������ �����                   *
+*  The subject is only required for the implementation of fishing       *
```
Comment (takhlaq) [inline src/map/items/item_fish.h]: fishing items

File: src/map/packets/inventory_item.cpp [hunk 4]
```diff
@@ -63,8 +63,8 @@ CInventoryItemPacket::CInventoryItemPacket(CItem* PItem, uint8 LocationID, uint8
 
                     uint32 CurrentTime = CVanaTime::getInstance()->getVanaTime();
 
-                    WBUFL(data,(0x15)) = ((CItemUsable*)PItem)->getNextUseTime();             // таймер следующего использования
-                    WBUFL(data,(0x19)) = ((CItemUsable*)PItem)->getUseDelay() + CurrentTime;  // таймер задержки использования
+                    WBUFL(data,(0x15)) = ((CItemUsable*)PItem)->getNextUseTime();             // next use the timer
```
Comment (takhlaq) [inline src/map/packets/inventory_item.cpp]: when we can next use this

File: src/map/packets/inventory_item.cpp [hunk 5]
```diff
@@ -63,8 +63,8 @@ CInventoryItemPacket::CInventoryItemPacket(CItem* PItem, uint8 LocationID, uint8
...
+                    WBUFL(data,(0x19)) = ((CItemUsable*)PItem)->getUseDelay() + CurrentTime;  // Use a timer delay
```
Comment (takhlaq) [inline src/map/packets/inventory_item.cpp]: when to finish animation i guess

File: src/common/socket.h [hunk 6]
```diff
@@ -152,16 +152,16 @@ extern time_t stall_time;
 
 int32 makeConnection(uint32 ip, uint16 port, int32 type);
 
-//��������� �������, �������������� ����������
+// function interface, connection processing
```
Comment (takhlaq) [inline src/common/socket.h]: handle sockets

**Comment by la0wai** [comment]
So should I remove them or edit them as @takhlaq's reviews say? :P

**Comment by teschnei** [comment]
remove em :)

**Pull request merged**